### ISSUE TITLE
FOB Capture  Unit Trasfer Update Request.

### ISSUE THREAD

**Issue by Blaze1969** [dcs-liberation/dcs_liberation#3334]
Asking that if you capture an enemy FOB that is not connected by a supply route to a friendly base via a successful air assault then the following turn allow transport mission to provide reinforcements via helicopter.  Or in other words I feel like you should be allowed to drop reinforcements  via transfer mission but the helicopters would need to RTB unless the FOB had a FARP/helipad.
Currently it appears this only possible with an airbase capture.

**Comment by Starfire13** [comment]
I have spoken to Blaze1969 on Discord about this. I think the current system is working as intended. Campaign designers already have control over whether a FOB can receive airlifted units by placing a FARP at that FOB.

**Comment by DanAlbert** [comment]
Oh, is that what this is about? FARPs can receive airlift units but FOBs can't? FOBs being airlift targets sounds reasonable to me, but it's unfortunately probably not as trivial as just changing a yes to a no, since FOBs don't actually have landing areas.

**Comment by Starfire13** [comment]
Yeah, I figured. It's one of those things that I think *could* be implemented but probably isn't worth doing right now, considering that:

1. This isn't an issue impacting any of the bundled campaigns, and is only an issue on a specific campaign the user has put together himself.
2. There is an easy and logical workaround for it (just adding a FARP to the FOB).
3. Adding this feature would decrease rather than increase the control a campaign designer potentially has over campaign flow.
